DeusEx/Classes/DXRFixup.uc

```diff
@@ -6,18 +6,30 @@ function FirstEntry()
     l( "mission " $ dxr.dxInfo.missionNumber $ " FirstEntry()");
 
     Level.AmbientBrightness += dxr.flags.brightness;
-
+    
     if( dxr.dxInfo.missionNumber == 6 )
         HongKong_FirstEntry();
-    if( dxr.dxInfo.missionNumber == 12 )
+    else if( dxr.dxInfo.missionNumber == 12 )
         Vandenberg_FirstEntry();
 }
 
 function AnyEntry()
 {
+    local DXRScopeView scope;
+    local DeusExRootWindow win;
+
     Super.AnyEntry();
     l( "mission " $ dxr.dxInfo.missionNumber $ " AnyEntry()");
 
+    win = DeusExRootWindow(dxr.Player.rootWindow);
+    if ( win.scopeView.IsA('DXRScopeView') == false ) {
+        scope = DXRScopeView(win.NewChild(Class'DXRScopeView', False));
+        scope.SetWindowAlignments(HALIGN_Full, VALIGN_Full, 0, 0);
+
+        scope.Lower();
+        win.scopeView = scope;
+    }
+
```

Comment:
This could maybe go into a "FixScopes" function, to keep AnyEntry fairly clean here (And align with FixUnbreakableCrates, and the rest of the per-mission stuff

Reply:
yea I'll do that, originally I didn't do it because I thought it would just be `win.scopeView = None;`